### ISSUE TITLE
Use GL_TIME_ELAPSED on OS X?

### ISSUE THREAD

**Issue by ghost** [num3ric/Cinder-Profiler#1]
I really haven't used `GL_ARB_timer_query` but not entirely for naught, [this tread on mac-opengl](https://www.mail-archive.com/mac-opengl@lists.apple.com/msg00001.html) seems to suggest that `GL_TIME_ELAPSED` is available for compatibility and core profile contexts… though it sounds as though the capabilities are more limited and the use a bit more constrained than `GL_TIMESTAMP`.

**Comment by num3ric** [comment]
Yeah, that's how it was before the last commit... I shouldn't have committed in master.

We're discussing using GL_TIME_ELAPSED as a fallback option but timestamp has net advantages: it can be nested and this logic will remain non-blocking even if the OpenGL command queue on the GPU is delayed by multiple frames.

I'm not alone here: https://answers.unrealengine.com/questions/222659/gpu-profiler-does-not-work-on-osxmac-os.html

**Comment by ghost** [comment]
Ahh no worries, I'll just use the previous rev for OS X support, thanks for the clarification!
